server/data/data-jdbc/src/test/java/org/apache/james/mailrepository/jdbc/JDBCMailRepositoryTest.java

```diff
@@ -79,21 +79,21 @@ private BasicDataSource getDataSource() {
 
     @Override
     @Disabled("JAMES-2546 This mail repository does not support null sender")
-    public void storeRegularMailShouldNotFailWhenNullSender() {
-
+    public void storeRegularMailShouldNotFailWhenNullSender() throws Exception {
+        MailRepositoryContract.super.storeRegularMailShouldNotFailWhenNullSender();
```

Comment:
If it's disabled it does not matter to have a body, as it will not be played. My previous comment was for the one with Unstable Tag, as those tests are still played, but in a separate phase of the build

Reply:
It does not hurt to have a body when disabled as far as I know

Reply:
It does not, just useless. But I don't mind TBH